<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>290</y>
     <width>251</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px;border-style:solid;border-color:black</string>
   </property>
   <property name="title">
    <string>Control Info</string>
   </property>
   <widget class="QLabel" name="label_ObjectName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::VLine</enum>
    </property>
    <property name="text">
     <string>ObjectName</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ObjectName">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>38</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_X">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Y">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_H">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_W">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_X">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Y">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Width">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_height">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_LineWidth">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Line Width</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_LineWidth">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SetFont">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="Barcode" name="widget_Barcode" native="true">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>480</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px;border-style:solid;border-color:black</string>
   </property>
  </widget>
  <widget class="MoveLabel" name="label_ItemBarcode">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>510</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Barcode</string>
   </property>
  </widget>
  <widget class="MoveLabel" name="label_ItemCode">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>540</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>ItemCode</string>
   </property>
  </widget>
  <widget class="MoveLabel" name="label_provider">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>540</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Provider</string>
   </property>
  </widget>
  <widget class="MoveLabel" name="label_unit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>520</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>ItemUnit</string>
   </property>
  </widget>
  <widget class="MoveLabel" name="label_Specs">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>580</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Specs</string>
   </property>
  </widget>
  <widget class="MoveLabel" name="label_PLU">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>560</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>PLU</string>
   </property>
  </widget>
  <widget class="MoveLabel" name="label_ProducingArea">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>570</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Producing area</string>
   </property>
  </widget>
  <widget class="MoveLabel" name="label_VIPPrice">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>480</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>VIP Price</string>
   </property>
  </widget>
  <widget class="MoveLabel" name="label_Sellprice">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>444</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sell Price</string>
   </property>
  </widget>
  <widget class="MoveLabel" name="label_ItemName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ItemName</string>
   </property>
  </widget>
  <widget class="MoveLabel" name="label_SpecialPrice">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>Special price</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>251</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px;border-style:solid;border-color:black</string>
   </property>
   <property name="title">
    <string>Label Info</string>
   </property>
   <widget class="QPushButton" name="pushButton_Save">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_LabelName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_LabelName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_ChooseLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>33</y>
      <width>72</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_ChooseLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_LabelWidth">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_LabelHeight">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_LabelWidth">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_LabelHeight">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_labels">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>569</width>
     <height>236</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px;border-style:solid;border-color:black</string>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
  </widget>
  <zorder>groupBox_labels</zorder>
  <zorder>groupBox</zorder>
  <zorder>widget_Barcode</zorder>
  <zorder>label_ItemBarcode</zorder>
  <zorder>label_ItemCode</zorder>
  <zorder>label_provider</zorder>
  <zorder>label_unit</zorder>
  <zorder>label_Specs</zorder>
  <zorder>label_PLU</zorder>
  <zorder>label_ProducingArea</zorder>
  <zorder>label_VIPPrice</zorder>
  <zorder>label_Sellprice</zorder>
  <zorder>label_ItemName</zorder>
  <zorder>label_SpecialPrice</zorder>
  <zorder>groupBox_2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MoveLabel</class>
   <extends>QLabel</extends>
   <header>MoveLabel.h</header>
  </customwidget>
  <customwidget>
   <class>Barcode</class>
   <extends>QWidget</extends>
   <header>Barcode.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
